components/resourceDetails/resourceSharing/agentAccessTable/index.test.jsx

```diff
@@ -125,7 +125,7 @@ const permissions = [
   },
 ];
 
-const permissionsWithConsentAgent = [
+const permissionsWithAccessAgent = [
```

Comment:
`permissionsWithAccessGrantBasedAccessAgent`

Reply:
This actually isn't being used anywhere so I took it out.